chord_metadata_service/mohpackets/apis/discovery.py

```diff
@@ -92,7 +92,8 @@ def count_donors(model: Type[Model], filters=None) -> Dict[str, int]:
 @discovery_router.get("/programs/", response=ProgramDiscoverySchema)
 def discover_programs(request):
     programs_list = Program.objects.values_list("program_id", flat=True)
-    return ProgramDiscoverySchema(cohort_list=programs_list)
+    metadata = Program.objects.values_list("metadata", flat=True)
+    return ProgramDiscoverySchema(cohort_list=programs_list, metadata=metadata)
```

Comment:
If you want to return metadata together with program_id as an object, you can use `programs = Program.objects.only("program_id", "metadata")`. 
But I'm not sure if you want a nested or flatted object in query service. Otherwise, it looks good!